README.md

```diff
@@ -1,11 +1,33 @@
-# aws-account-operator
+# AWS-ACCOUNT-OPERATOR
+
+
+## General Overview
 
 This operator will be responsible for creating and maintaining a pool of AWS accounts. The operator will create the initial setup and configuration of the those accounts, IAM resources and expose credentials for a IAM user with enough permissions to provision a 4.0 cluster.
 
-The operator requries a secret named `aws-account-operator-credentials` containing credentials to the AWS payer account you wish to create accounts in. The secret should contain credentials for an IAM user in the payer account with the data fields `aws_access_key_id` and `aws_secret_access_key`. The user should have the following IAM permissions:
+
+The operator is deployed to an openshift cluster. It will create a pool of AWS accounts and secrets containing IAM credentials inside of a namespace. The operator will then wait for a accountClaim to be created in any namespace. It will then tie that accountClaim to an account and credentials previously created, and it will put required infromation into the namespace where the accountClaim was created.
+
+
+On our hive clusters there is a namespace called `aws-account-operator`. This is where the aws-account-operator runs and where the account CRs and the IAM user secrets are created.
```

Comment:
I think this line can be removed (assuming my suggestion above and below in the Requirements section are accepted)